quickstarts/delphix/config.yml

```diff
@@ -0,0 +1,39 @@
+# Name of the quickstart (required)
+name: Delphix
+
+# Displayed in the UI (required)
+title: Delphix
+
+# Long-form description of the quickstart (required)
+description: |
+  This Delphix quickstart allows you to get visibility into all your Delphix Virtualization platforms. Use this quickstart together with this [integration](https://github.com/delphix/dct-newrelic-integration) 
+
+# Displayed in search results and recommendations. Summarizes a quickstarts functionality.
+summary: |
+  Monitor Delphix Virtualization operations with New Relic.
+
+# Support level: New Relic | Verified | Community (required)
+level: Community
+
+# Authors of the quickstart (required)
+authors:
+  - Diego Loureda
+  - Carlos Cuellar
+
+# Keywords for filtering / searching criteria in the UI
+keywords:
+  - delphix
+  - dct
+  - vdb
+  - dsource
+  - "data control tower"
```

Comment:
```suggestion
```

Reply:
what are the `dct, vbd, dsource` keywords for? We maintain a pretty strict list of keywords since this is used in our navigation and search so unless these are standard industry terms I'm leaning towards removing them.

Reply:
These are Delphix platform concepts , here I share with you more details about them:
- DCT= Data Control Tower - https://docs.delphix.com/dctmc
- VDB = Virtual Database. - https://docs.delphix.com/docs/overview-of-data-virtualization/glossary-of-major-delphix-concepts
- dsource  - https://docs.delphix.com/docs/overview-of-data-virtualization/glossary-of-major-delphix-concepts

DCT can be redundant so i think we can remove it but we'll prefer to keep the other ones.